Makefile

```diff
@@ -79,7 +79,7 @@ WARNFLAGS= -Wall -Wsign-compare
 CFLAGS = -DMSHADOW_FORCE_STREAM $(WARNFLAGS)
 
 ifeq ($(DEV), 1)
-	CFLAGS += -g -Werror
+	CFLAGS += -g -Werror -fno-var-tracking-assignments
```

Comment:
Can't we try to use a #pragma directly on the code instead of disabling for all platforms in the makefile :-(  ?